.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
I also not like the use of `.gitmodules`, but want to have on end a complete own development kit, which can used with ways like this.
- As build and system install inside Kodi Itself, to have e.g. on linux a `/usr/lib/kodi-addon-sharedlibrary-api2.a`
- As build with Kodi's add-on steps
- As build if nowhere available on add-on where it becomes handled on his depends

With the last step you can create addon without the use anything else from Kodi.